src/lib.rs

```diff
@@ -31,6 +31,7 @@ extern crate multipart;
 extern crate base64;
 extern crate flate2;
 extern crate serde;
+pub extern crate chrono;
```

Comment:
I think it would be better to let users depend on the chrono crate directly, rather than re-exposing it, because it is a nonessential part of `discord-rs`'s functionality.